### DISCUSSION TITLE
Effort for Edge2 Support?

### DISCUSSION THREAD

**Discussion by castlec** [Joshua-Riek/ubuntu-rockchip#535] [Q&A]
I feel like this is this is going to be my OS between my devices going forward. There's a community armbian build by balbes150. He has stopped doing 5.10 kernel builds and has been focused on mainline. I'd like the consistency of using this build with my OP5s and Edge2s. Would there be more effort than utilizing existing edge2 device overlays? Do the supported devices differ in more ways than the overlays and kernel command lines?

**Comment by Joshua-Riek** [comment]
I do not have the Edge 2 device, so I have no idea what the software status of the Edge 2 is. I don't understand your question.

**Reply by castlec** [reply]
It's slightly better than the OP5 from a vendor perspective. The major difference is that you're supporting the OP5 and other rk3588 devices. The community consensus is that your builds are the best for these devices.  You put a lot of effort into it and it shows. I'm asking, conceptually, how hard it would be to support the edge2. I imagine the best case is take an existing device overlay and build away. I'm willing to contribute to your codebase if I'm not off the mark and can do so with testing of builds and creation of images for that testing if necessary. If I'm utterly off the mark, let me know and I'll forget about it.
Thanks for your efforts and contributions to the ecosystem. 

Edit: if you believe that putting a device in your hands is the best way to go about it, I'm willing to make that happen.

**Reply by Joshua-Riek** [reply]
I'd need the device to do any sort of development, I've reached out to Khadas in the past and never got a response. I can reach out again and maybe they will respond this time.
